lib/assets/stylesheets/modules/_page-review.scss

```diff
@@ -1,13 +1,13 @@
 .page-expiry--not-expired,
 .page-expiry--default {
-  padding: $gutter-half;
-  margin-top: $gutter;
-  margin-bottom: $gutter;
-  background-color: $govuk-blue;
-  color: $page-colour;
+  padding: govuk-spacing(3);
```

Comment:
The page review panel is missing font styles, and the expired panel now has a grey background.

## Before

<img width="1485" alt="Screen Shot 2019-09-04 at 11 08 00" src="https://user-images.githubusercontent.com/121939/64246200-47077280-cf04-11e9-857c-bd8ee8804a0e.png">

## After

<img width="1485" alt="Screen Shot 2019-09-04 at 10 51 20" src="https://user-images.githubusercontent.com/121939/64246171-3a831a00-cf04-11e9-99d6-4e803a7c82f9.png">